docs/cli/synth.md

```diff
@@ -152,3 +152,39 @@ You can also control the plugins storage directory by one of:
 
 See [cdk8s-validation-plugin-example](https://github.com/cdk8s-team/cdk8s-validation-plugin-example)
 
+## Synthesize cdk8s app as helm chart
+
+You can synthesize your cdk8s application to a format that is compatible with [helm](https://helm.sh/). 
+
+!!! note
+  You will need helm to be [installed](https://helm.sh/docs/intro/install/) locally for this feature.
+
+For example, you can run:
+
+```
+cdk8s synth --format helm --chart-version 1.0.0 --chart-api-version v2
+```
+
+Then it will synthesize a generated helm chart to the output folder(default is 'dist'). The manifest synthesized will be placed in `templates/` folder. If any crds were mentioned in `cdk8s.yaml` config file as `imports`, then they will be downloaded and placed in `crds` folder. 
```

Comment:
```suggestion
Then it will synthesize a generated helm chart to the output folder (default is 'dist'). The synthesized manifests will be placed in `templates/` folder. If any crds were mentioned in `cdk8s.yaml` config file as `imports`, they will be downloaded and placed in the `crds/` folder. 
```